src/main/java/hanieum/conik/ConikApplication.java

```diff
@@ -3,9 +3,11 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.cloud.openfeign.EnableFeignClients;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
 
 @EnableFeignClients
 @SpringBootApplication
+@EnableJpaAuditing
```

Comment:
이것때문에 현재 테스트 시에 에러가 나는 것 같은데, 이건 유지하고
ProposalOrphanRemovalTest 파일에서 
클래스에 붙은 @EnableJpaAuditing 이거 없애고 아래처럼 작성하면 잘 통과할거에요

```suggestion
@SpringBootTest
@Transactional
class ProposalOrphanRemovalTest {

    @Autowired
    private EntityManager em;

    @TestConfiguration
    @EnableJpaAuditing
    static class JpaAuditingTestConfig {}
```